mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -640,11 +640,13 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
                     handled = true
                 }
                 R.id.default_sitemap -> {
-                    val sitemap = serverProperties?.sitemaps?.firstOrNull { s -> s.name == prefs.getDefaultSitemap() }
+                    val sitemap = serverProperties?.sitemaps?.firstOrNull {
+                            s -> s.name == prefs.getDefaultSitemap(connection)
```

Comment:
`s ->`  should be on old line + indent by 4 spaces

(here and below)

Reply:
Done